tests/homepage.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('specific viewport block', () => {
+  test.use({ viewport: { width: 1600, height: 1200 } });
+
+  test('has title', async ({ page }) => {
+    await page.goto('http://localhost:8080');
```

Comment:
If a `baseURL` is added to the playwright.config, could update these calls to use relative urls: 
```suggestion
    await page.goto('/');
```

Reply:
Implemented :)